zscaler/help.md

```diff
@@ -5,19 +5,20 @@
 # Key Features
 
 * Lookup categories for a given URL
+* Add or remove URLs from blacklist
 
 # Requirements
 
 * [Requires a Zscaler organization API Key](https://help.zscaler.com/zia/api-getting-started#RetrieveAPIKey)
 * Requires a Zscaler username and password
 
-
 # Documentation
+
 ## Setup
 
 To locate your base URI and key:
 
-1. Log in to the ZIA Admin Portal using your admin credentials. 
+1. Log in to the ZIA Admin Portal using your admin credentials.
```

Comment:
Is this the same URL for any customer? Ie. the URL doesn't change depending on the organization or anything? If so, can you link that URL here?

Reply:
For now is 5 different URLs. Unfortunately I'm not found a way to make it easier for a customer (only an enum, but if they add some URL it will make some problems).

Reply:
https://help.zscaler.com/zia/api-getting-started
In section "Retrieve your base URI and API key" :)